pkg/serializer/jsonstream/payload_builder.go

```diff
@@ -37,8 +37,27 @@ func NewPayloadBuilder() *PayloadBuilder {
 	}
 }
 
+// OnErrTooBigPolicy defines the behavior when OnErrTooBig occurs.
+type OnErrTooBigPolicy int
+
+const (
+	// ContinueOnErrTooBig: when founding an errTooBig, skips the error and continue
+	ContinueOnErrTooBig OnErrTooBigPolicy = iota
+
+	// FailedErrTooBig: when founding an errTooBig, returns the error and stop
+	FailedErrTooBig
+)
+
 // Build serializes a metadata payload and sends it to the forwarder
 func (b *PayloadBuilder) Build(m marshaler.StreamJSONMarshaler) (forwarder.Payloads, error) {
+	return b.BuildWithOnErrTooBigPolicy(m, ContinueOnErrTooBig)
```

Comment:
Nit: `ErrTooBig` is a bit obscure from outside this package IMO. What do you think about renaming it `ErrItemTooBig` ?